docs/source/schema/directives.md

```diff
@@ -1,40 +1,73 @@
 ---
-title: Using schema directives
-description: Using schema directives to transform schema types, fields, and arguments
+title: Directives
+sidebar_title: Directives
+description: Configure schema types, fields, and arguments
 ---
 
-A _directive_ is an identifier preceded by a `@` character, optionally followed by a list of named arguments, which can appear after almost any form of syntax in the GraphQL query or schema languages. Here's an example from the [GraphQL draft specification](http://facebook.github.io/graphql/draft/#sec-Type-System.Directives) that illustrates several of these possibilities:
+A **directive** decorates part of a GraphQL document with additional configuration. Tools like Apollo Server (and [Apollo Client](https://www.apollographql.com/docs/react/local-state/managing-state-with-field-policies/#querying)) can read a GraphQL document's directives and perform custom logic as appropriate.
 
-```typescript
+Directives are preceded by the `@` character, like so:
+
+```graphql{2}
+type ExampleType {
+  oldField: String @deprecated(reason: "Use `newField`.")
+  newField: String
+}
+```
+
+This example shows the `@deprecated` directive, which is a [default directive](#default-directives) (i.e., it's part of the [GraphQL specification](http://spec.graphql.org/June2018/#sec--deprecated)). It demonstrates the following about directives:
+
+* Directives can take arguments of their own (`reason` in this case).
+* Directives appear _after_ the declaration of what they decorate (the `oldField` field in this case)
+
+## Valid locations
+
+Each directive's definition specifies _where_ it can appear in a GraphQL document. For example, here's the GraphQL spec's definition of the `@deprecated` directive:
```

Comment:
You should probably explain somewhere that directives have definitions, which are part of the schema; a directive needs to be defined in the schema in order for you to use it in that schema or in an operation being validated against the schema.